client/lib/create-selector/README.md

```diff
@@ -7,11 +7,11 @@ This module exports a single function which creates a memoized state selector fo
 
 `createSelector` accepts the following arguments:
 
-- A function which calculates the cached result given a state object and any number of variable arguments necessary to calculate the result
-- A function or array of functions which return array of dependent state, given the state and the same arguments as the selector
+- A function which selects data from the state. Given a state object and any number of other arguments it calculates a result, which is cached for later reuse.
+- A function which returns the part or parts of the state tree the selector depends on. This can be a single value or an array.
 - _(Optional)_ A function to customize the cache key used by the inner memoized function
 
-For example, we might consider that our state contains post objects, each of which are assigned to a particular site. Retrieving an array of posts for a specific site would require us to filter over all of the known posts. While this would normally be an expensive operation, we can use `createSelector` to create a memoized function:
+For example, our state contains post objects, each of which is assigned to a particular site. To retrieve an array of posts for a specific site, we'd have to filter all of the known posts. While this would normally be an expensive operation, we can use `createSelector` to create a memoized function:
```

Comment:
I can see some slight room for improvement here too, though I am nitpicking here:

"While this would normally be an expensive operation"

It categorically **_is_** an expensive operation and caching doesn't solve that. To me, this sentence seems to suggest that it does. What we're trying to convey is that repeated use avoids having to recompute that expensive op.